dvc/fs/gdrive.py

```diff
@@ -109,11 +109,14 @@ def _validate_config(self):
         if (
             self._use_service_account
             and not self._service_account_json_file_path
+            and "GDRIVE_CREDENTIALS_DATA" not in os.environ
```

Comment:
```suggestion
            and self.GDRIVE_CREDENTIALS_DATA not in os.environ
```

Reply:
In the previous codes, it is `GDriveFileSystem.GDRIVE_CREDENTIALS_DATA`, better to be consistent?